apps/homelab/app/globals.css

```
@@ -25,8 +25,7 @@
   --font-inter:
     "Inter", -apple-system, BlinkMacSystemFont, ui-sans-serif, system-ui,
     sans-serif;
-  --font-serif:
-    "Libre Baskerville", Georgia, serif;
+  --font-serif: "Libre Baskerville", Georgia, serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)